libraries/AP_ESCSerialTelem/AP_ESCSerialTelem.cpp

```diff
@@ -143,5 +143,70 @@ bool HWESC_Telem::parse_packet(void)
     return true;
 }
 
-#endif // HAL_PERIPH_ENABLE_HWESC
+#if AP_ESC_SERIAL_TELEM_ESC_TELEM_BACKEND_ENABLED
+const AP_Param::GroupInfo AP_ESCSerialTelem_ESCTelem_Backend::var_info[] = {
```

Comment:
this conflicts with the ESC_NUMBER parameter in AP_Periph

Reply:
ahh, I see, we're only instantiating the inner class, not the class with parameters